### ISSUE TITLE
When accessibility is on, semantics debugger fails to get updates

### ISSUE THREAD

**Issue by Hixie** [flutter/flutter#1752]
You get a blank screen, and then you can no longer tap on anything.

**Comment by goderbauer** [comment]
I cannot reproduce this. However, there is issue https://github.com/flutter/flutter/issues/10566, which is similar to this one, but it is present regardless of accessibility being on or off.
